<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulate</class>
 <widget class="QDialog" name="Simulate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1142</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulate</string>
  </property>
  <widget class="QLabel" name="label_mean_norm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Укажите матожидание</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Не выбрано</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Гауссовское (нормальное)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Показательное (экспоненциальное)</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="Summary">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Задайте вид распределения погрешности:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="disp_norm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="mean_norm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Симуляция</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="plotnos" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>731</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Гистограмма погрешности</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_disp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Укажите СКО</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lambda_exp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_exp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Укажите лямбду</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_prop">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>470</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вероятность появления погрешности</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prop">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="calc_prop">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>comboBox</zorder>
  <zorder>Summary</zorder>
  <zorder>pushButton</zorder>
  <zorder>plotnos</zorder>
  <zorder>label_5</zorder>
  <zorder>disp_norm</zorder>
  <zorder>label_disp</zorder>
  <zorder>mean_norm</zorder>
  <zorder>label_mean_norm</zorder>
  <zorder>lambda_exp</zorder>
  <zorder>label_exp</zorder>
  <zorder>label_prop</zorder>
  <zorder>prop</zorder>
  <zorder>calc_prop</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
